packages/debugger/src/panels/kernelSources/filter.tsx

```diff
@@ -8,23 +8,30 @@ import React from 'react';
 import { IDebugger } from '../../tokens';
 
 import { Search } from '@jupyter/react-components';
+import { ITranslator, nullTranslator } from '@jupyterlab/translation';
 
 /**
  * The class name added to the filebrowser crumbs node.
  */
 export interface IFilterBoxProps {
   model: IDebugger.Model.IKernelSources;
+
+  /**
+   * The application language translator
+   */
+  translator?: ITranslator;
 }
 
 const FilterBox = (props: IFilterBoxProps) => {
+  const trans = (props.translator ?? nullTranslator).load('jupyterlab');
```

Comment:
Tiny optimization suggestion: to avoid loading the translator again you could instead pass the translation bundle (`trans`) in the props.

Reply:
Adapted the comments.